xcode/Subconscious/Shared/Services/DatabaseService.swift

```diff
@@ -554,17 +583,18 @@ final class DatabaseService {
         )
         return results.compactMap({ row in
             guard
-                let address = row.col(0)?
+                let did = row.col(0)?.toString()?.toDid(),
+                let address = row.col(1)?
                     .toString()?
-                    .toLink()?
```

Comment:
Why was this step removed?

Reply:
Restored it, but answered here: https://github.com/subconsciousnetwork/subconscious/pull/925#discussion_r1353825665